confgenerator/confgenerator.go

```diff
@@ -395,12 +396,16 @@ func (l *Logging) generateFluentbitComponents(userAgent string, hostInfo *host.I
 
 func generateSeveritySelfLogsParser() []fluentbit.Component {
 	out := make([]fluentbit.Component, 0)
+
+	_, timeOffset := time.Now().Zone()
+
 	parser := LoggingProcessorParseRegex{
 		Regex:       `(?<message>\[[ ]*(?<time>\d+\/\d+\/\d+ \d+:\d+:\d+)] \[[ ]*(?<severity>[a-z]+)\].*)`,
 		PreserveKey: true,
 		ParserShared: ParserShared{
 			TimeKey:    "time",
 			TimeFormat: "%Y/%m/%d %H:%M:%S",
+			TimeOffset: fmt.Sprintf("%d", timeOffset),
```

Comment:
In addition to the above, I also think we should omit this if it's zero to simplify the diff.